src/components/htmlvideoplayer/plugin.js

```diff
@@ -1591,6 +1595,39 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
         return false;
     };
 
+    HtmlVideoPlayer.prototype.isAirPlayEnabled = function () {
+
+        if (document.AirPlayEnabled) {
+            return document.AirplayElement ? true : false;
+        } else {
+            var video = this._mediaElement;
+            if (video) {
+                return video.webkitShowPlaybackTargetPicker();
```

Comment:
I'm not sure we want this else statement since this method is only used to check the AirPlay status, not trigger a selection menu.

Reply:
Will adjust. It was a late night :-p